### DISCUSSION TITLE
CVE vulns from RedHat base image plan

### DISCUSSION THREAD

**Discussion by mkgeotab** [keycloak/keycloak#22987] [Q&A]
Hey Keycloak team!

The keycloak image seems to be based off of ubi9-micro -> https://github.com/keycloak/keycloak/blob/main/quarkus/container/Dockerfile

I saw there was a handful of security vulnerabilities that affect RHEL 9 but they don't seem to have plans to fix them. WIll Keycloak team look to address the affected libraries in their own image?

References:
https://access.redhat.com/security/cve/cve-2020-12413
https://access.redhat.com/security/cve/cve-2023-2603
https://access.redhat.com/security/cve/cve-2023-2602

Thanks so much in advance!

**Comment by vmuzikar** [comment]
Please see our [guide](https://www.keycloak.org/security.html) on how to report security issues. Thanks.

**Reply by abstractj** [reply]
@vmuzikar for public known CVEs we can discuss them publicly, there's no need to use the keycloak-security mailing list.
